ZAPD/ZRoom/Commands/SetActorList.cpp

```diff
@@ -104,7 +98,7 @@ string SetActorList::GenerateSourceCodePass2(string roomName, int baseAddress)
 	}
 
 	zRoom->parent->AddDeclarationArray(
-		segmentOffset, DeclarationAlignment::None, DeclarationPadding::Pad16, actors.size() * 16,
+		segmentOffset, DeclarationAlignment::None, DeclarationPadding::None, actors.size() * 16,
```

Comment:
Is this true for OoT and MM?

Reply:
I assume OOT actor lists were the end of a separate .c file, so they were padded out to 16-bytes. In MM, it seems some other structs could be at the end of them, so this padding isn't always correct.

Reply:
I'm not sure if the padding is really needed in OoT, but in the meanwhile you could add a game check.
```c
        DeclarationPadding padding = DeclarationPadding::Pad16;
        if (Globals::Instance->game == MM_RETAIL)
        {
            padding = DeclarationPadding::None;
        }
        zRoom->parent->AddDeclarationArray(
		segmentOffset, DeclarationAlignment::None, padding, actors.size() * 16,
```